tests/bleTestsScripts/testClass.py

```diff
@@ -438,6 +444,23 @@ def advertisement(testDevice, DUT_UUID=None):
 
         return True
 
+    @staticmethod
+    def get_manufacture_data(testDevice, DUT_UUID=None):
+        manufacture_data_dict = bleAdapter.getPropertie(testDevice, "ManufacturerData")
+
+        # If manufacture data doesn't exist, return None
+        if( manufacture_data_dict == None ):
+            print("No Manufacture Data")
+            sys.stdout.flush()
+            return None
+
+        # If manufacture data exists, return manufacture data
+        else:
+            print( manufacture_data_dict.items() )
```

Comment:
Include some additional string as to what is this log.

Reply:
changed it